english/golden-lab/golden-lab.md

```diff
@@ -0,0 +1,5 @@
+#Golden Lab
+
+You walk past a few dogs; one was sleeping, one was a little too aggressive, and one was just plain ugly. However, shortly you notice a beautiful Golden Lab. You immediately fall in love with him because you notice he's giving you a "can you believe this guy?" look while nodding his head towards a poodle in the corner. The poodle is wearing an umbrella hat.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.